### DISCUSSION TITLE
defer not working on select element

### DISCUSSION THREAD

**Discussion by Bradzer** [livewire/livewire#4326] [1. Help]
The `defer` modifier seems not to work on my `<select>` element. Selecting a value updates the page and does not defer until next action. Here's the code made shorter for convenience:

**Component usage**
```blade
<LivewireComponent>
    <x-inputs.select-picker name="agency_id" class="select2" deferMode="defer" data-placeholder="Selectionner un magasin" label="Magasin" required />
</LivewireComponent>
```

**Component definition**
```blade
@props(['required', 'name', 'label', 'placeholder' => null, 'deferMode' => false])

<div class="form-group">
    <label for="{{ $name }}">{{ __($label) }} @isset($required) <x-required></x-required> @endisset</label>
    
    @if ($deferMode)
        <select wire:model.defer="{{ $name }}" id="{{ $name }}" {{ $attributes->merge(['class' => 'form-control '. validate($errors, $name)]) }} x-ref="{{ $name }}">
    @else
        <select wire:model.debounce.500ms="{{ $name }}" id="{{ $name }}" {{ $attributes->merge(['class' => 'form-control '. validate($errors, $name)]) }} x-ref="{{ $name }}">
    @endif
        
        @if(!$attributes->has('multiple'))
            <option value="">{{ $placeholder ?? "Selectionner un(e) ". $label }}</option>
        @endif
        {{ $slot }}
    </select>
    {!! validate_feedback($errors, $name) !!}
</div>
```
Is there anything wrong here ?

PS: `agency_id` is indeed a public property in the Livewire component class.

**Additional info**
Livewire version:  2.8.1

**Comment by joshhanley** [comment]
@Bradzer please don't keep opening issues as we are no longer using them, we are only using discussions.

And please be patient, we try out best to get to any opened discussions but that can take time. If you don't get a response in a few days, add a comment to your discussion to bump it up the list.

I've just tested your select component and defer in your component seems to be working fine for me, see my below screencapture of your code running.

![recording (60)](https://user-images.githubusercontent.com/882837/145140369-5726194d-5de6-44d7-8eea-a2e5fdcf0e32.gif)

Can you share your component class and view so we can see how you have it configured, and also your full selected blade component if you stripped it down?

Thanks!
